case-server/src/main/java/com/powsybl/caseserver/parsers/EntsoeFileNameParser.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.caseserver.parsers;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.entsoe.util.EntsoeGeographicalCode;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A utility class to work with Entsoe file name parsers.
+ *
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@AutoService(FileNameParser.class)
+public class EntsoeFileNameParser implements FileNameParser {
+
+    public static final Pattern FILE_NAME_REGEX = Pattern.compile("(\\d{8}[_]\\d{4})[_](\\w{3})[_](\\w{3}).*");
+    public static final String DATE_FORMAT = "yyyyMMdd_HHmm";
+
+    public static DateTime parseDateTime(String dateStr) {
+        DateTimeFormatter df = DateTimeFormat.forPattern(DATE_FORMAT);
+        return df.withZone(DateTimeZone.forID("Europe/Brussels")).parseDateTime(dateStr);
+    }
+
+    @Override
+    public boolean exists(String fileBaseName) {
+        return FILE_NAME_REGEX.matcher(fileBaseName).matches();
+    }
+
+    @Override
+    public EntsoeFileName parse(String fileBaseName) {
```

Comment:
Why you did not rely on powsybl-core EntsoeFileName class? Or improve it if necessary?